content/en/platform/corda/5.0/deploying-operating/config/secrets.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Configuration Secrets"
+date: '2023-05-12'
```

Comment:
in some of these pages you're missing `version: ` front matter field

Reply:
My understanding is that files will take this from the _index file in the same bundle.